### PR TITLE
feat: add folders script (#24)

### PR DESCRIPTION
**Pull request by aninda052**

### CODE REVIEW

**Comment by foyzulkarim** [comment]
does it work in windows?

**Comment by aninda052** [comment]
no vaia.
 i'll refactor the code for wondows.

File: server/src/zx.mjs
Comment (foyzulkarim) [inline server/src/zx.mjs:1]: very interesting work. can you please change the file name to something more meaningful? :) 
we can reuse this file to run more system commands in future.
eg. `shell-script.mjs` can be a good option

and also, try to run the same command multiple times to see if the successive runs fail the execution or not. 

awesome work 👍

Comment (aninda052) [reply server/src/zx.mjs:1]: vai, i've checked multiple times, the command runs perfectly on windows 10. I'm running windows 10 on virtualBox.

**Comment by foyzulkarim** [comment]
Hi @aninda052 
i was thinking, should we keep it inside of the `src` folder? it is not a business or application related code. I think we should move it out. 
You can create a folder under `server` as `scripts` and move the file into there.

Now I remember, we may also need to run `migrations` later. Those scripts can also be stored into the `scripts` folder. 
what do you think?

**Comment by aninda052** [comment]
> Hi @aninda052 i was thinking, should we keep it inside of the `src` folder? it is not a business or application related code. I think we should move it out. You can create a folder under `server` as `scripts` and move the file into there.
> 
> Now I remember, we may also need to run `migrations` later. Those scripts can also be stored into the `scripts` folder. what do you think?

@foyzulkarim vai, as `shell-script` is not a application related code, i think we should move it from `src` folder. so "create a folder under `server` as `scripts` and move the file into there" would be nice idea. 

Btw, what kind of `migrations` you are referring to? can you give me a bit details ?

**Review by foyzulkarim — APPROVED** [review]
Nice work!

**Comment by aninda052** [comment]
> Nice work!

Thank you :)

**Pull request merged**